app/routes/projectHandlers.js

```diff
@@ -27,7 +27,7 @@ module.exports = function (projectHelpers, markdownDirectory) {
 
     var compileMarkdown = function compileMarkdown(req, res, next){
         var projectName = req.params.projectName;
-        var fpath = path.join(markdownDirectory, projectName + '.md'); //fname = ${projectName}.md TODO - confirm this is always true
+        var fpath = path.join(markdownDirectory, projectName);
```

Comment:
This change caused this route https://github.com/drayside/se-capstone-db/blob/1-add-command-line-arg-for-directory-path/server.js#L79 to fail, now when I remove out the `+ .md ` the route works. But another route https://github.com/drayside/se-capstone-db/blob/1-add-command-line-arg-for-directory-path/server.js#L63 fails. I can't understand how the first route (first link) can cause compileMarkdown to be executing. I'm perplexed.